packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -1425,6 +1508,7 @@ tests:
           javascript: Date('Sun, 17 Dec 1995 03:24:00 GMT')
           shell: Date('Sun, 17 Dec 1995 03:24:00 GMT')
           object: Date('Sun, 17 Dec 1995 03:24:00 GMT')
+          php: (new \DateTime('1995-12-17 03:24:00', new \DateTimeZone('UTC')))->format('D M d Y H:i:s e')
```

Comment:
It was made by the csharp example: 
ToString("ddd MMM dd yyyy HH':'mm':'ss UTC")
Remake it?

Reply:
Let's use [`DateTimeInterface::RFC3339_EXTENDED`](https://www.php.net/manual/en/class.datetimeinterface.php#datetime.constants.rfc3339_extended).

That seems the most concise and portable, and will also include milliseconds in the output (relevant since BSON's date type includes milliseconds).

Reply:
Fixed.

Reply:
OK to resolve.